hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterClockImpl.java

```diff
@@ -46,4 +47,16 @@ public void setMasterTime(long masterTime) {
     public long getClusterTimeDiff() {
         return (clusterTimeDiff == Long.MAX_VALUE) ? 0 : clusterTimeDiff;
     }
+
+    @Override
+    public long getClusterStartTime() {
```

Comment:
Is this local time? What if clocks of nodes are not sync? What if some of the nodes are in different time-zone?